pkg/updater/updater.go

```diff
@@ -1082,6 +1082,9 @@ func ConstructGrid(log logrus.FieldLogger, cols []InflatedColumn, issues map[str
 	}
 
 	for _, col := range cols {
+		if col.Column != nil {
```

Comment:
Is it worth putting this new behavior behind a flag?

Reply:
I was debating on it, but yeah might as well!

Reply:
Is this a new field? If so then maybe not so important. Does it even appear in the UI?

Reply:
But still IMO good practice!~

Reply:
New field, doesn't appear in the UI yet (though maybe it'll be a nice FixIt item to flag columns as broken, not quite trivial due to how column headers get combined rn). Given this is a shared updater function, would it make sense to flag this in tabulator:main + use `brokenThreshold > 0.0` explicitly here?

Reply:
Sure, the goal here is mostly to design the failure, maybe when we encounter "this is for mysterious reasons causing testgrid to crash" or "all the updaters are surprisingly slow after this push" that we have a cheap way to turn this functionality off while we investigate.